plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -258,7 +258,7 @@
 
     private String _updateHostPasswdPath;
 
-    private int _dom0MinMem;
+    private long _dom0MinMem;
```

Comment:
I see no reason not to default it to something sane.
Wido - 
The way it is setup right now, admins would not be able to change it to something lower than whatever we default to unless still greater than 768mb or 10% of system ram.  At least not by specifying it via the host.reserved.mem.mb property.  That being said, the default should be greater than 768mb.  A 1G default sounds sane to me.